### DISCUSSION TITLE
Apply 'Default Recipe Preferences' changes to existing recipes

### DISCUSSION THREAD

**Discussion by epheterson** [mealie-recipes/mealie#1415] [Feature Request]
**Before submitting this feature request I have**
- [ X ] Reviewed the documentation and verified that my feature request isn't already a feature.
- [ X ] Reviewed the existing feature requests and my feature has not been requested.
- [ X ] Checked the [tasks taged](https://github.com/hay-kot/mealie/issues?q=is%3Aissue+is%3Aopen+label%3Atask+) issues and verified my feature is not covered

**Please Describe The Problem To Be Solved**
When modifying 'Default Recipe Preferences' within 'Group Settings' the changes apply only to newly-created recipes. In many cases, you'd want these changes to apply to *all* recipes, which is a very manual process.

**(Optional): Suggest A Solution**
It'd be great if there were a checkbox or confirmation pop-up on each change to choose whether you'd like changes to apply to only new recipes, or all existing recipes.

Another option is a bulk editor which allows changing these values. There is a bulk editor in Data Management, though it doesn't expose these preferences.

**Additional Information**
- [ ] If the feature is accepted I'm willing to submit a PR to provide this feature

**Comment by hay-kot** [comment]
Makes sense to me. Tracking Here

- https://github.com/hay-kot/mealie/issues/1416
